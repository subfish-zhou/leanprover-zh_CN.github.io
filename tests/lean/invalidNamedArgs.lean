
def ex1 (xs : List Nat) : Nat :=
xs.foldl (b := 0) fun sum x => sum + x

def f (a : Nat) (flag := true) : Nat :=
a + if flag then 1 else 0

def g (a : Nat) : Nat :=
f a (flg := false)
