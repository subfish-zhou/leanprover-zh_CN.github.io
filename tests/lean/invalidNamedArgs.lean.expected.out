invalidNamedArgs.lean:3:9-3:17: error: invalid argument name 'b' for function 'List.foldl'
invalidNamedArgs.lean:9:4-9:18: error: invalid argument name 'flg' for function 'f'
